src/app/header/header.component.scss

```diff
@@ -0,0 +1,26 @@
+#logo-img {
+  object-fit: cover;
+  margin: 1px;
+  width: 60px;
+  height: 60px;
+}
+
+.header-container {
+  width: 100%;
+  //max-height: 80px;
+  height: 10vh;
+  background: var(--dark-blue);
+  margin: 0;
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  font-size: 24px;
+  font-family: Roboto, sans-serif;
+  color: var(--main-white);
+}
+
+#calculate-link {
+  font-size: 20px;
+  text-decoration: none;
+  color: var(--main-white);
+}
```

Comment:
For styling - it's best to use `#id` as a last resort CSS selector.
> attributes > classes > siblings/children > id

is a good rule of thumb when choosing a selector.
Also, remember that all selectors within a components `*.scss` file are scoped to that component, so in here, you could just target `img`, instead of `#logo-img`.